cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,21 +491,6 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
```

Comment:
Here is a reference, and my memory is slowly returning. It has to do whether or not native methods are being used for focus (I had stepped through a lot of this code in the debugger and saw where native methods end up getting called). With native methods being used, the correct thing can't happen in a unit test.

https://bugs.jquery.com/ticket/13363